src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -243,6 +237,12 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                         services.AddSingleton<WorkspaceDirectoryPathResolver, DefaultWorkspaceDirectoryPathResolver>();
                         services.AddSingleton<RazorComponentSearchEngine, DefaultRazorComponentSearchEngine>();
 
+                        if (configure != null)
+                        {
+                            var builder = new RazorLanguageServerBuilder(services);
+                            configure(builder);
+                        }
+
```

Comment:
I had to do this otherwise everything that the configure does is overridden by the defaults.

Reply:
Hmm, can you validate that the "Report a Problem" feature in VS still uploads Razor logs with Data? This is how that plugs in

Reply:
Just validated it using the exp instance. It still uploads the logs.

Reply:
Did the logs have data in them?

Reply:
The Razorlogs one did have data. I didn't check the other ones.

Reply:
That's sufficient thanks for validating.